nautobot/core/templates/components/panel/body_content_text.html

```diff
@@ -1,10 +1,14 @@
 {% load helpers %}
-{%  if render_as == render_options.json %}
+{%  if render_as == 'json' %}
     {{ text_content | render_json }}
```

Comment:
Do we need to render "not-pretty" json? Just highlighted?
![image](https://github.com/user-attachments/assets/44a96e66-78d2-4d11-afa4-083da6003c88)